### PR TITLE
Add patch for doxy2swig compilation issue. 

### PR DESCRIPTION
**Pull request by dvs999**
Patch taken from Fedore Core 32 SRPM

### CODE REVIEW

File: sphinxbase/default/PKGBUILD [hunk 1]
```diff
@@ -5,23 +5,27 @@
 
 pkgname=sphinxbase
 pkgver=5prealpha
-pkgrel=11
+pkgrel=12
```
Comment (Martchus) [inline sphinxbase/default/PKGBUILD:8]: People who already have the package do not need to rebuild it so don't increase the `pkgrel` here.

File: sphinxbase/default/PKGBUILD [hunk 2]
```diff
@@ -5,23 +5,27 @@
...
 license=('BSD')
 makedepends=('bison' 'swig')
 depends=('python2' 'python' 'lapack' 'libpulse') # not sure if libsamplerate is needed 'libsamplerate'
-source=("http://downloads.sourceforge.net/project/cmusphinx/${pkgname}/${pkgver}/${pkgname}-${pkgver}.tar.gz")
-sha256sums=('f72bdb59e50b558bed47cc2105777200d2b246a0f328e913de16a9b22f9a246f')
+source=("http://downloads.sourceforge.net/project/cmusphinx/${pkgname}/${pkgver}/${pkgname}-${pkgver}.tar.gz"
+        "sphinxbase-5prealpha-fix-doxy2swig.patch")
+
+sha256sums=('f72bdb59e50b558bed47cc2105777200d2b246a0f328e913de16a9b22f9a246f'
+	    '1cb485202f83dc517872f5ab41f59d18884af1b85799166d80e08860f7729919')
```
Comment (Martchus) [inline sphinxbase/default/PKGBUILD:19]: The indentation is messed here.

**Comment by dvs999** [comment]
Should be resolved

**Comment by Martchus** [comment]
Thanks. I squashed the commit via GitHub.

**Pull request merged**